modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFieldValuesProviderTest.java

```diff
@@ -114,12 +114,8 @@ public static void tearDownClass() {
 	public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 
-		ServiceContext serviceContext =
-			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
-
-		serviceContext.setRequest(_getHttpServletRequest(_group));
-
-		ServiceContextThreadLocal.pushServiceContext(serviceContext);
```

Comment:
@vngoliveira we still have the `ServiceContextThreadLocal.popServiceContext()` in the `tearDown` method. But you removed the `ServiceContextThreadLocal.pushServiceContext(serviceContext)` from the  `setUp` method.

That said, we also need remove the `ServiceContextThreadLocal.popServiceContext()` from the `tearDown` method and change the test to something like:

```
ServiceContextThreadLocal.pushServiceContext(serviceContext);

try {
   // all the logic
}
finally {
   ServiceContextThreadLocal.popServiceContext();
}

```

Reply:
This way we guarantee that only what was "pushed" will be removed from the stack.